src/build/env.py

```diff
@@ -118,14 +126,17 @@ def install(self, requirements):  # type: (Iterable[str]) -> None
         """
         Install packages from PEP 508 requirements in the isolated build environment.
 
-        :param requirements: PEP 508 requirement specification to install
+        :param requirements: PuP 508 requirement specification to install
```

Comment:
Find and replace gone awry? ;)

Reply:
Oops, thought I corrected that! VI user error. :)